trace/Sampling.md

```diff
@@ -0,0 +1,32 @@
+# Sampling
+
+This document is about the sampling bit, sampling decision, samplers and how and when does 
+OpenCensus sample traces. A sampled traces is a traces that gets exported via the configured
+exporters.
+
+## Sampling Bit (propagated via TraceOptions)
+
+Sampling bit is always set only at the start of a Span, and is using a `Sampler`
+
+### What kind of samplers does OpenCensus support?
+* `AlwaysSample` - sampler that always makes a "yes" decision every time.
```

Comment:
PropagateDecision? it seems we don't so maybe we need a sampler that does?

Reply:
So I see AlwaysSample and NeverSample just as samplers to control specific requests/paths. Usually as default samplers we will use something like the ProbabilitySampler which will propagate the decision.

PropagateDecision only can be achieved with a ProbabilitySampler(0.0).